gen3-integration-tests/tests/test_gen3_workflows.py

```diff
@@ -271,7 +271,7 @@ def test_happy_path_create_tes_task(self):
                 {
                     "image": "public.ecr.aws/docker/library/alpine:latest",
                     "command": [
-                        f"cat /data/input.txt > /data/output.txt && grep hello /data/input.txt > /data/grep_output.txt && echo '{echo_message}'",
+                        f"cat /data/input.txt > /data/output.txt && grep hello /data/input.txt > /data/grep_output.txt && echo {echo_message}",
```

Comment:
I would argue we should leave the quotes in so we can catch that Funnel issue. Quotes should be allowed. If we remove them for now to unblock the tests, let's add a TODO to explain the situation.